.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
   Test:
     executor:
       name: ios/default
-      xcode-version: "12.0"
+      xcode-version: "12.3.0"
```

Comment:
Use Xcode 12.3, so we can run on the latest iOS Simulators – 14.3.

Using the same version between local development and CI is helpful because there are quirks in UI tests between iOS Simulator versions.